frontend/packages/console-app/src/components/volume-snapshot/create-volume-snapshot/create-volume-snapshot.tsx

```diff
@@ -133,18 +130,10 @@ const isDefaultSnapshotClass = (volumeSnapshotClass: VolumeSnapshotClassKind) =>
   ] === 'true';
 
 const CreateSnapshotForm = withHandlePromise<SnapshotResourceProps>((props) => {
-  const {
-    resourceName,
-    plural,
-    namespace,
-    kindObj,
-    handlePromise,
-    inProgress,
-    errorMessage,
-  } = props;
+  const { pvcName, namespace, handlePromise, inProgress, errorMessage } = props;
 
   const { t } = useTranslation();
-  const [pvcName, setPVCName] = React.useState(resourceName);
+  const [statePvcName, setPVCName] = React.useState(pvcName);
```

Comment:
The setter function name should match the variable name. Also note that acronyms should be capitalized like `statePVCName` by the conventions in the console repo.

Can we give something more descriptive than `statePvcName`? Something that describes how this is different than the `pvcName` which is passed in through props?

Reply:
Well, it's quite hard to answer, it's really the same thing except one is props (provided by parent) and one is state (mutated inside component). If you want I can rename the state one `finalPvcName`, but don't hesitate to tell me if you got a better idea

Reply:
done